MySchedule/MyScheduleApp.swift

```diff
@@ -14,7 +14,7 @@ struct MyScheduleApp: App {
         let schema = Schema([
             Task.self
         ])
-        let modelConfiguration = ModelConfiguration(schema: schema, isStoredInMemoryOnly: false)
```

Comment:
WARNING - Operators should be surrounded by a single whitespace when they are being used